src/webgpu/api/validation/encoding/beginComputePass.spec.ts

```diff
@@ -106,6 +106,40 @@ g.test('timestampWrites,query_set_type')
     t.tryComputePass(isValid, descriptor);
   });
 
+g.test('timestamp_query_feature')
+  .desc(
+    `
+  Test that a validation error is generated if the features doesn't contain 'timestamp-query'
+  feature.
```

Comment:
Description also needs to describe what call is generating the error; see below.